debian/eos-install-app-helper.install

```diff
@@ -0,0 +1,7 @@
+usr/bin/*
+usr/share/eos-application-tools/icon-overrides/*
```

Comment:
It would be nice to list these in alphabetical order.  Also, did you intentionally omit the usr/share/doc directory that is in the current package?

Reply:
That's added in a later step by the Debian stuff, those files aren't from the "upstream" build.

Reply:
Okay, thanks.